src/app/auth/profiles/profiles.component.ts

```diff
@@ -118,6 +119,10 @@ export class ProfilesComponent implements OnInit {
    * @param profile - Profile selected by user.
    */
   selectProfile(profile: Profile): void {
```

Comment:
Currently, When you click on `None`, it also throws the same error message here... 

You might want to have a look at this also!

Reply:
good catch!

Reply:
Nice! I see that you have fixed it :smile: